script/elements.js

```diff
@@ -0,0 +1,9 @@
+export const createTaskButtonElement = document.getElementById("create-task");
+export const createTaskCardElement =
+    document.getElementById("task-card-create");
+export const createTaskInputElement =
+    document.querySelector(".task-card__input");
+export const addTaskButtonElement = document.getElementById("add-task-button");
```

Comment:
what's the difference between `add task` and `create task` ?

Reply:
changed the name